npc/re/merchants/barters.yml

```diff
@@ -44,3 +44,7 @@
 Header:
   Type: BARTER_DB
   Version: 1
+
+Footer:
+  Imports:
+  - Path: npc/re/merchants/eden_market.yml
```

Comment:
I would prefer to add the file in a new folder, "barters", in which we will put the future shops

Reply:
inside merchants folder?

Reply:
Yes, like https://github.com/rathena/rathena/pull/6550/files#diff-607908e542d38c82fbc390b1efc36bd2588581731e27a9ae33b8d45d954547faR50